src/ClamavValidator/ClamavValidatorException.php

```diff
@@ -0,0 +1,16 @@
+<?php namespace Sunspikes\ClamavValidator;
+
+use Exception;
+
+class ClamavValidatorException extends Exception
+{
+    /**
+     * @param string $message
+     * @param integer $code
+     * @param Exception $previous
+     */
+    public function __construct($message = '', $code = 0, Exception $previous = null)
```

Comment:
I think you don't need an explicit constrictor here, because it doesn't add anything...